src/Razor/test/testapps/AppWithP2PReference/AppWithP2PReference.csproj

```diff
@@ -5,18 +5,14 @@
   </PropertyGroup>
 
   <PropertyGroup>
-    <DisableImplicitFrameworkReferences>true</DisableImplicitFrameworkReferences>
-    <AddRazorSupportForMvc>true</AddRazorSupportForMvc>
+    <TargetFramework>netcoreapp3.0</TargetFramework>
   </PropertyGroup>
 
   <ItemGroup>
-    <FrameworkReference Include="Microsoft.NETCore.App" />
+    <!-- Avoid referencing the AspNetCore framework since we want the ability to target work-in-progress shims -->
+    <FrameworkReference Remove="Microsoft.AspNetCore.App" />
```

Comment:
I would like to understand how this works.

Reply:
We're letting the WebSDK and .NET do the default which would include references to NETCore.App and AspNetCore.App. We remove this to avoid picking up older builds of the AspNetCore fx.